2_Dictionaries.ipynb

```diff
@@ -167,7 +167,7 @@
   },
```

Comment:
<p>is every letter in SONG separated by a new line? </p><p>you have successfully decrypted the letters but completely changed the format of the encrypted message - this means your decryption is deprecated :) fix it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week04-ofirshan/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='118'/>

Reply:
<p>fixed</p>